tests/behat/theme_boost_union_contentsettings_staticpages.feature

```diff
@@ -190,25 +190,33 @@ Feature: Configuring the theme_boost_union plugin for the "Static pages" tab on
   @javascript
   Scenario Outline: Setting: Static page page title - Set a custom static page page title
     Given the following config values are set as admin:
-      | config             | value              | plugin            |
-      | enable<page>       | yes                | theme_boost_union |
-      | <page>content      | <p>Lorem ipsum</p> | theme_boost_union |
-      | <page>pagetitle    | Custom             | theme_boost_union |
+      | config          | value                                                                                             | plugin            |
+      | enable<page>    | yes                                                                                               | theme_boost_union |
+      | <page>content   | <p>Lorem ipsum</p>                                                                                | theme_boost_union |
+      | <page>pagetitle | <span lang="en" class="multilang">Custom</span><span lang="de" class="multilang">Angepasst</span> | theme_boost_union |
```

Comment:
While this enhanced test is perfectly fine, I feel tempted to leave the existing test as it is and to move the multilanguage feature into a dedicated test. In this case, you could also add test steps for configuring multilanguage static page _content_.

What do you think?

Reply:
If you follow the tradition of Moodle core, those multilang tests are in the existing ones.

Reply:
So, if test steps for configuring multilanguage static page content are to be included in the existing ones, fine. But not separate ones please.

Reply:
Ah, I was not aware that Moodle core builds multilang tests that way. In this case, I'm with you.

Regarding static page content, I will check if I can sneak some additional steps in in a separate PR.